server/plugin/command.go

```diff
@@ -367,7 +367,18 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.BotUserID,
+		Message:   msg,
+	}
+
+	if _, err := p.API.CreatePost(post); err != nil {
+		p.API.LogWarn("error while creating post", "post", post, "error", err.Error())
```

Comment:
I don't think we should pass in the `Post` itself here. Is that something you've seen elsewhere @sibasankarnayak?

```suggestion
		p.API.LogWarn("error while creating post", "err", err.Error())
```

Reply:
@mickmister yes in github there are lots of reference for this like https://github.com/mattermost/mattermost-plugin-github/blob/4f633dada08971478c184723dd5b7340ff98979d/server/plugin/webhook.go#L326 and 			p.API.LogWarn("Error webhook post", "post", post, "error", err.Error())